gemrb/includes/Platform.h

```diff
@@ -0,0 +1,50 @@
+/* GemRB - Infinity Engine Emulator
+ * Copyright (C) 2003 The GemRB Project
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ *
+ *
+ */
+
+#ifndef PLATFORM_H
+#define PLATFORM_H
+
+#ifdef HAVE_CONFIG_H
+	#include <config.h>
+#endif
+
+#include "exports.h"
+
+#ifndef _MAX_PATH
+	#ifdef WIN32
+		#define _MAX_PATH 260
+	#else
+		#define _MAX_PATH FILENAME_MAX
+	#endif
+#endif
+
+#ifdef WIN32
+	#include "win32def.h"
+#elif defined(HAVE_UNISTD_H)
+	#include <unistd.h>
+#endif
+
+#include <cstdio>
```

Comment:
Nice, no more including VFS.h everywhere. Once we fully move to nullptr, cstdio can be inlined as well. :)